app/controllers/webhooks_controller.rb

```diff
@@ -200,6 +208,10 @@ def handle_payment_intent_succeeded(
             }
           )
           save_payment_intent = true
+          WebhookManager::CrawlReceiptCreator.call(
```

Comment:
Is there not one place at the bottom of the webhook that we can put this to consolidate these three calls into one?

Reply:
Or, if this is called in the majority, reverse the logic to only _not_ call this in the opposite case